infrastructure/terraform/modules/data-store/iam-binding.tf

```diff
@@ -24,24 +24,46 @@ resource "google_project_iam_member" "email-role" {
   project = data.google_project.data_processing.project_id
 }
 
+locals {
+  dataform_sa = "service-${data.google_project.data_processing.number}@gcp-sa-dataform.iam.gserviceaccount.com"
+}
+
+# When the service account is not created, gcloud returns NOT_FOUND
+# When the service account is created, gcloud returns IAM_PERMISSION_DENIED
+# Because it's error messages we pipe the stderr to stdout
+resource "null_resource" "wait_for_dataform_sa_creation" {
+  provisioner "local-exec" {
+    command = <<-EOT
+    while true; do
+      output=$(gcloud iam service-accounts describe ${local.dataform_sa} 2>&1)
+      if [[ $output == *IAM_PERMISSION_DENIED* ]]; then
```

Comment:
Why this condition? Shouldn't it match something positive. Also, need timeout and failure.